src/plugins/intel_gpu/src/plugin/ops/fully_connected.cpp

```diff
@@ -26,22 +26,24 @@ namespace ov {
 namespace intel_gpu {
 
 static void CreateFullyConnectedCompressedOp(ProgramBuilder& p, const std::shared_ptr<op::FullyConnectedCompressed>& op) {
-    validate_inputs_count(op, {4, 5});
+    validate_inputs_count(op, {4, 5, 6});
     auto inputs = p.GetInputInfo(op);
     std::string primitive_name = layer_type_name_ID(op);
     auto supports_immad = p.get_engine().get_device_info().supports_immad;
 
-    const int INPUT_CNT_WITH_ZP = 5;
     auto input_name = inputs[0].pid;
     auto weights_name = inputs[1].pid;
     auto bias_name = inputs[2].pid;
     auto scale_name = inputs[3].pid;
-    auto zp_name = inputs.size() == INPUT_CNT_WITH_ZP ? inputs[4].pid : "";
+    size_t input_idx = 4;
+    const size_t INPUT_PORT_IDX = input_idx;
+    std::string zp_name = op->get_has_zp() ? inputs[input_idx++].pid : "";
+    std::string activation_scale_name = op->get_has_activation_scale() ? inputs[input_idx++].pid : "";
```

Comment:
I suppose you can use `inputs[...]` directly w/o `.pid` to avoid explicit out port set below

Reply:
For conditional statement, if I don't use `pid` it fails as below.

```
/home/mingyuki/local/ov1/src/plugins/intel_gpu/src/plugin/ops/fully_connected.cpp:40:44: error: operands to ‘?:’ have different types ‘__gnu_cxx::__alloc_traits<std::allocator<cldnn::input_info>, cldnn::input_info>::value_type’ {aka ‘cldnn::input_info’} and ‘const char [1]’
   40 |     std::string zp_name = op->get_has_zp() ? inputs[input_idx++] : "";
      |                           ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Because the result is not string, but `input_info`

Reply:
applied, thanks!